src/Recruit.Vacancies.Client/Infrastructure/Client/VacancyClient.cs

```diff
@@ -12,24 +14,25 @@ namespace Esfa.Recruit.Vacancies.Client.Infrastructure.Client
     public class VacancyClient : IVacancyClient
     {
         private readonly IMessaging _messaging;
-        private readonly IQueryStoreReader _queryStore;
+        private readonly IQueryStoreReader _reader;
         private readonly IVacancyRepository _repository;
 
-        public VacancyClient(IVacancyRepository repository, IQueryStoreReader queryStore, IMessaging messaging)
+        public VacancyClient(IVacancyRepository repository, IQueryStoreReader reader, IMessaging messaging)
         {
             _repository = repository;
-            _queryStore = queryStore;
+            _reader = reader;
             _messaging = messaging;
         }
 
-        public Task UpdateVacancyAsync(Vacancy vacancy)
+        public async Task UpdateVacancyAsync(Vacancy vacancy)
         {
             var command = new UpdateVacancyCommand
             {
                 Vacancy = vacancy
             };
 
-            return _messaging.SendCommandAsync(command);
+            await _messaging.SendCommandAsync(command);
+            await UpdateDashboardAsync(vacancy.EmployerAccountId);
```

Comment:
Should we update the dashboard every time we update a vacancy?  Maybe we should ass in a bool so the orchestrators can specify that they have changed a field that affects the dashboard. Dunno.  One for Mark.

Reply:
I am not sure, it's difficult because this could easily be forgotten about in other places that might need it. I don't really like having to do these two things but that is what has been suggested.

Reply:
if we did add a flag then i'd say it should update by default

Reply:
so `bool forceUpdate = true` as the last parameter?

Reply:
updateQueryStores?